File: public/phase-2-20160620022203/C_SHARP/arny-s-workout.cs
// Name : Arny’s Workout
using System;
namespace Application {
	class arny’s_workout {
		public static void Main(string[] args) {
		// input:
		// n :  the number of exercises
		// a :  the number of repeats for the exercises
		int index;
		int n = Convert.ToInt32(Console.ReadLine());
		int[] a = new int[n];
		string[] a_elements = (Console.ReadLine()).Split(' ');
		for(index=0;index<n;index++)
			a[index] = Convert.ToInt32(a_elements[index]);


		// write your code here
		// store your results in `result`

		// output
		// Dummy Data
		string result="chest";

		Console.WriteLine(result);

		}	
	}
}

File: public/phase-2-20160620022203/C/arny-s-workout.c
// Name : Arny’s Workout
#include <stdio.h>
int main() {
	// input:
	// n :  the number of exercises
	// a :  the number of repeats for the exercises
	int n, a[20], index;
	char result[105]="chest";
	scanf("%d", &n);
	for(index = 0; index< n; index++) {
		scanf("%d", &a[index]);
	}


	// write your code here
	// store your results in `result`

	// output
	printf("%s", result);
	return 0;
}